examples/repl/repl.cpp

```diff
@@ -0,0 +1,60 @@
+#include <kalcy/kalcy.hpp>
+#include <cassert>
+#include <format>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+
+namespace {
+struct Repl {
+	const std::string_view EXIT_TOKEN{"exit"};
```

Comment:
`static constexpr`. (`const` members break assignment.)

Reply:
And not screaming case, instead `exit_token_v` or similar.

Reply:
I have removed the token as a variable entirely after the last commit and am instead comparing `text` to the literal value.